core/templates/services/math-interactions.service.ts

```diff
@@ -65,6 +65,10 @@ export class MathInteractionsService {
         'Your answer seems to be missing a variable/number after the "' +
         errorMessage[0] + '".');
     }
+    if (errorMessage === 'Not a prefix operator.') {
+      errorMessage = (
+        'Your answer seems to be missing a number before the × operator.');
```

Comment:
Is this error message guaranteed to occur only in the × case, or can it also occur in the division etc. case?

Notice how the other error messages (like the one below) check for the operator that is being used in the expression string so that they can report the right thing to the user.